common/app/layout/Sublink.scala

```diff
@@ -352,12 +352,15 @@ case class PaidCard(
   image: Option[ImageMedia],
   fallbackImageUrl: Option[String],
   targetUrl: String,
+  cardTypes: Option[ItemClasses] = None,
   branding: Option[Branding]
 ) extends FaciaCard
 
 object PaidCard {
 
-  def fromPressedContent(content: PressedContent): PaidCard = {
+  def fromPressedContent(content: PressedContent): PaidCard = this.fromPressedContent(content, None)
```

Comment:
No need for a second functions: You can simply declare a second argument with a default value.